packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -28,14 +29,21 @@
   private final FlutterWebViewClient flutterWebViewClient;
   private final Handler platformThreadHandler;
 
+  @TargetApi(Build.VERSION_CODES.JELLY_BEAN_MR1)
   @SuppressWarnings("unchecked")
   FlutterWebView(
-      Context context,
+      final Context context,
       BinaryMessenger messenger,
       int id,
       Map<String, Object> params,
       final View containerView) {
+
+    DisplayListenerProxy displayListenerProxy = new DisplayListenerProxy();
```

Comment:
The code in that class is a huge workaround, and we know we only need it for WebView `58.0.3029.125` and below. Do you think it's worth checking what webview is actually running with `WebViewCompat.getWebViewClient` and only initializing this where we need it for the bugfix?

Reply:
I actually rather use this workaround widely so that potential issues are easier to track. I'm currently not aware of any issues that should be caused by the workaround.